tests/test_cli.py

```diff
@@ -92,6 +96,34 @@ def test_run_on_repo(update_header, is_excluded, check_output, updated):
     assert update_header.call_count == 2
 
 
+@mock.patch('subprocess.check_output')
+@mock.patch('unbeheader.cli.is_excluded')
+@mock.patch('unbeheader.cli.update_header')
```

Comment:
just wondering, why not use `pytest-mock` for this?

Reply:
Right. Let's do this in the future.